Annual review of SOP0007: GDI-SOP0007_sop-template-creation.md (#70)
GenomicDataInfrastructure/standard-operating-procedures: sops/european-level/GDI-SOP0007_sop-template-creation.md

### 3. Roles and Responsibilities
See the qualifications and responsibilities of the roles at the [**Organisational Roles and Responsibilities**](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_organisational-roles-and-responsibilities.md) document.

| Role       | Full name       | GDI/node role   | Organisation | Notes |
|------------|-----------------|-----------------|--------------|-------|
| Author     | Marcos Casado Barbero | Task 4.3 member | EMBL-EBI |       |
| Reviewer   | Pedro Ferreira | Task 4.3 member | IST |       |
| Reviewer   | Bianca Hendriksze | Task 4.3 member | HRI |       |
| Reviewer   | Elisavet Torstensson | Task 4.3 member | UU / NBIS |       |
| Reviewer   | Mattias Strömberg | Task 4.3 member | UU / NBIS |       |
| Approver   | Óscar Martínez | SDPC member | CRG |       |
| Approver   | Gabriele Rinck |  | EMBL-EBI |       |

### 4. Purpose

> This SOP will help maintainers of the [``GenomicDataInfrastructure/standard-operating-procedures``](https://github.com/GenomicDataInfrastructure/standard-operating-procedures) repository, as well as GDI's Operations Committee (OC), to **transform SOP Requests into SOP templates**. The OC manages the entire development process, with the Security and Data Protection Committee (SDPC) participating in the approval phase. In doing so, the process will be reproducible and straightforward, ensuring consistency and quality in SOP development across the GDI project.

Reviewer: What about this text instead:"This SOP defines the process of creating an SOP, from the initial SOP request (through the [New SOP Request] to a fully authorised, released SOP. This will help everyone involved in the process, such as the maintainers of the [GenomicDataInfrastructure/standard-operating-procedures] repository, members GDI's Operations Committee (OC) or those involved in drafting, reviewing and approval. In doing so, the process will be reproducible and straightforward, ensuring consistency and quality in SOP development across the GDI project."

Author: Adapted

> ### 5. Scope

Reviewer: what about this text for the scope section: "This SOP breaks the process of creating an SOP down into detailed steps covering all stages from the initial SOP request by any member of GDI, review of the request, the crucial role of GDI’s Operations Committee in steering the SOP creation process, the selection of authors and reviewers as well as the approval and authorisation of the SOP prior to release of the SOP. The output of this SOP will be the finalised document added to the repository."

Regardless of the format, **fill out the draft with as much information** (e.g., background, purpose, summary...) **as possible**, to the best of your knowledge. This content will be finished by the authors later on.

Proceed to ⏩[step 3](#83-contact-authors).

#### 8.3. Contact Authors
| Step identifier | When| Who |
|:----------------|:----|:----|
| ``3`` | After SOP document has been drafted | OC |

> Once the SOP document has been drafted, experts are required to fill in the gaps and finalise it. These **authors are to be appointed and contacted by you, as part of the OC**. Who the authors are will depend on the background and requirements of each SOP, and thus it is your responsibility as an OC member to **find the best-suited people for the task**. The only requirements are for authors to be part of the GDI project and to know about, or be part of, the subject the SOP revolves around. Alternatively, you may contact the GDI Work Package (WP) leaders to point you in the right direction.

Reviewer: Instead of "Once the SOP document has been drafted," I would suggest "Once a basic draft of the SOP has been prepared,"

We hope this message finds you well.

> The GDI's Operations Committee (OC) has drafted a new Standard Operating Procedure (SOP): "< SOP Title >". We now seek your expertise to fill in the gaps and help us finalise it. Given your background and involvement in the GDI project, we believe you are well-suited as an author of the SOP.

Reviewer: instead of "has drafted a new Standard Operating Procedure (SOP)" I would suggest "has prepared a basic draft for a new Standard Operating Procedure (SOP)"

< Your Name >
< Your GDI Node >
Operations Committee (OC)
````

Depending on the format of the draft, the above-mentioned ``< URL of drafted document >`` will change:
- **Google Document**. You can simply paste the URL of the Google Document, making sure to add the recipients as editors of that particular document in Google Drive.
- **Markdown document**. The easiest way to share the document would be by creating a drafted PR in GH, from either a personal fork or branch to the ``dev`` branch. Make sure to create it as a _Draft pull request_. If done this way, at [step 6](#86-create-pr-review-and-merge) you will simply have to convert the draft into a _Ready for review_ PR. Check the [accessioning guide](../../docs/GDI-SOP_sop-accessioning.md#file-naming-conventions) to know how to name and where to place this new file.

Remember to **leave a comment in the GitHub issue**, briefly mentioning that authors have been contacted. Be mindful of the information you share (e.g., no email addresses), since _anything_ that is in this GitHub repository will be **publicly displayed**!

> This step of the process ends when enough GDI members accepted authoring the SOP. For this to happen, it may be needed for you, as an OC member, to engage in conversations to find the best-suited authors. These conversations will be useful when reviewers are needed.

Reviewer: ```suggestion
This step of the process ends when enough GDI members accepted authoring the SOP. For this to happen, it may require you, as an OC member, to engage in conversations to find the best-suited authors. These conversations will also be useful for the selection of reviewers (step 4.1 below).
```

Proceed to ⏩[step 4](#84-monitor-sop-development).

#### 8.4. Monitor SOP development
| Step identifier | When| Who |
|:----------------|:----|:----|
| ``4``                  | After SOP authors have been appointed | OC |

It is your duty, as a member of the OC, to monitor the entire SOP development process, ensuring that:
- **Authors** are engaged with the development of the SOP. Checking the content of the SOP or contacting the authors recurrently may be required.

> - **Reviewers and approvers are appointed**, diverse across GDI nodes, and engaged in the process. Beyond the role definitions in the [ORR](../../docs/GDI-SOP_organisational-roles-and-responsibilities.md), the rule of thumb is for these roles to span multiple GDI Pillars and nodes. Reviewers may be selected directly by the OC or by the authors themselves. The OC and SDPC appoint their respective approval positions.

Reviewer: ```suggestion
- **Reviewers and approvers are appointed**, diverse across GDI nodes, and engaged in the process. Beyond the role definitions in the [ORR](../../docs/GDI-SOP_organisational-roles-and-responsibilities.md), the rule of thumb is for these roles to span multiple GDI Pillars and nodes, as appropriate. Reviewers may be selected directly by the OC or by the authors themselves. The OC and SDPC appoint their respective approval positions.
```

- Every **gate-keeping event** (review, approval, authorisation) **is documented in the GH issue**. The OC should record reviewer and approver appointments (e.g., ``Reviewed by ...; pending approval by ...``), the committee represented by each approver, any delegation and its responsible delegator, the two-week approval deadlines, and all approval or justified rejection outcomes.

In each of the subsequent sub-steps, back and forth communication between all actors may be needed to address requests.

Proceed to ⏩[step 4.1](#841-appoint-reviewers).

##### 8.4.1 Appoint reviewers
| Step identifier | When| Who |
|:----------------|:----|:----|
| ``4.1``         | After SOP authors have been appointed | OC |

Similar to how authors were nominated and contacted, you shall **appoint reviewers and contact them** requesting their review. Depending on the SOP, who are the reviewers and how to contact them will vary. If by email, you can use the email template below.

> Remember to CC the OC and SDPC mailing lists: ``gdi_operations_committee [at] elixir-europe.org`` and ``gdi_security_data_protection [at] elixir-europe.org``.

Reviewer: ```suggestion
Remember to CC the OC mailing list: ``gdi_operations_committee [at] elixir-europe.org`` 
```

Reviewer: I don't think we need to loop the SDPC in at this step!??

| ``4.2``         | After reviewers have completed SOP review | OC and SDPC |

Once the drafted SOP has been filled in by the authors and has passed the inspection of reviewers, it shall go through approval by the OC and SDPC. The following rules apply:

- **Each committee appoints two approval positions** for the SOP. Commonly, this results in **four** distinct approvers.
- A person who is a **member of both committees** may be appointed by both and approve on behalf of both. In this case, one additional approver must be appointed by each committee, resulting in **three distinct approvers**.
- A **reviewer** may also be appointed as an approver of the same SOP.
- Each committee may use a **rota to share approval appointments evenly** among its members.
- An appointed OC or SDPC member may **delegate the approval task** to another GDI member, including someone who is not a committee member. The delegation fulfils the delegator's committee approval position, but the **delegating member retains responsibility** and accountability for that approval. Both the delegate and responsible delegator must be documented in the GH issue and the Roles and Responsibilities section of the SOP.

> If an approver delegates on someone else, both _Approver_ and _Delegate_ need to be properly added to the SOP's Roles and Responsibilities table. This includes recording the delegation in both roles' _Notes_ cells. For example, `Delegated approval to X` (at Approver's) and `Approval delegated by Y` (at Delegate's).

Reviewer: ```suggestion
If an approver delegates to someone else, both _Approver_ and _Delegate_ need to be properly added to the SOP's Roles and Responsibilities table. This includes recording the delegation in both roles' _Notes_ cells. For example, `Delegated approval to X` (at Approver's) and `Approval delegated by Y` (at Delegate's).
```

< Your Name >
< Your GDI Node >
Operations Committee (OC)
````

This **step shall finish** either:

> - When the **period to raise issues (4 full weeks) has concluded without any complaints from the MB**.

Reviewer: ```suggestion
- When the **period to raise issues (4 full weeks) has concluded without any objections from the MB**.
```
